templates/partials/answers/checkbox.html

```diff
@@ -1,10 +1,12 @@
 {% from "components/checkboxes/_macro.njk" import onsCheckboxes %}
 
+{% set number_of_checkboxes = answer.options | length %}
+
 {{ onsCheckboxes({
   "legend": question_title,
   "id": answer.id,
   "legendClasses": "u-vh",
-  "checkboxesLabel": '' if answer.label == '' else answer.label or _("Select all that apply"),
+  "checkboxesLabel": '' if number_of_checkboxes == 1 else answer.label or _("Select all that apply"),
```

Comment:
I think if label is provided it should be used, regardless of the number of checkboxes.

Reply:
I've updated the logic so the answer.label should always be used